FRESH_START_AND_RESTART.s

```diff
@@ -932,7 +932,7 @@ DUMMYAD		EXIT
 		
 # Page 200
 		TC	BANKCALL
-		CADR	UPACTOFF		# TURN OFF UPLINK ACTIV LIGHT
+		CADR	UPACTOFF		# TURN OFF UPLINK ACTIVE LIGHT
```

Comment:
`ACTIV` is used in [source](http://www.ibiblio.org/apollo/ScansForConversion/Comanche055/0200.jpg)